ui/desktop/README.md

```diff
@@ -106,7 +100,7 @@ These environment variables may be used to customized the build.
 | `APP_UPDATER_CURRENT_VERSION` | Version of client. |
 | `APP_UPDATER_LATEST_VERSION_TAG` | Next version for comparison with current version. |
 | `APP_UPDATER_LATEST_VERSION_LOCATION` | Location of app release to use for updating client. Can be a filepath or url. |
-| `BYPASS_CLI_SETUP` | Disable download and extraction of cli. For development use only. |
+| `CLI_SETUP` | Enable download and extraction of CLI. For development use only. |
```

Comment:
Do you think this should be called `SETUP_CLI` instead since it's nicer to read with the verb in the beginning?

Is it still considered `"For development use only."` or should we remove that now?

Reply:
Absolutely agree. `SETUP_CLI` sounds and read way better. I will provide the change.

Yes!!! God catch, I will delete the "for dev only"